clang/lib/CodeGen/CodeGenFunction.h

```diff
@@ -5148,6 +5151,9 @@ class CodeGenFunction : public CodeGenTypeCache {
                            SourceLocation ArgLoc, AbstractCallee AC,
                            unsigned ParmNum);
 
+  /// EmitWriteback - Emit callbacks for function.
+  void EmitWritebacks(CodeGenFunction &CGF, const CallArgList &args);
```

Comment:
nit: 
```suggestion
  void EmitWritebacks(CodeGenFunction &CGF, const CallArgList &Args);
```
https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly